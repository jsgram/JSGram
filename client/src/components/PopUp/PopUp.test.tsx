import { PopUpModal, IProps } from './index';
import { shallow } from 'enzyme';
import React from 'react';

describe('PopUpModal component:', () => {
    let renderer: any;

    beforeEach(() => {
        const props: IProps = {
            loading: true,
            modal: true,
            photo: 'cat',
            deletePhoto: jest.fn(() => 'some value'),
            toggleModal: jest.fn(() => 'some value'),
        };
        renderer = shallow(<PopUpModal {...props} />);
    });

    test('render - success', () => {
        expect(renderer).toMatchSnapshot();
    });
});
