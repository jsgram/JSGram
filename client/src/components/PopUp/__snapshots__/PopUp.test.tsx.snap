// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopUpModal component: render - success 1`] = `ShallowWrapper {}`;
